plugins/@grouparoo/sqlite/__tests__/table-import/import-property.ts

```diff
@@ -0,0 +1,806 @@
+          },
+          [{ op, key: "id", match: "15" }]
+        );
+        expect(value).toEqual(1);
+      });
+      test("string", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "purchase", match: "Apple" }]
+        );
+        expect(value).toEqual(2);
+      });
+      test("string is case sensitive", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "purchase", match: "apple" }]
+        );
+        expect(value).toEqual(0);
+      });
+      test("date", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "date", match: "2020/02/15" }]
+        );
+        expect(value).toEqual(1);
+      });
+      test("timestamp", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "stamp", match: "2020/02/15 12:13:14" }]
+        );
+        expect(value).toEqual(1);
+      });
+      test("float", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "amount", match: "1.54" }]
+        );
+        expect(value).toEqual(2);
+      });
+    });
+
+    describe("greater than", () => {
+      const op = "greater than";
+      test("integer", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "id", match: "15" }]
+        );
+        expect(value).toEqual(2);
+      });
+      test("string", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "purchase", match: "Apple" }]
+        );
+        expect(value).toEqual(4);
+      });
+      test("string is case sensitive", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "purchase", match: "apple" }]
+        );
+        expect(value).toBeGreaterThanOrEqual(0); // unpredictable ascii math
+      });
+      test("date", async () => {
+        const value = await getPropertyValue(
+          {
+            column,
+            sourceMapping,
+            aggregationMethod,
+          },
+          [{ op, key: "date", match: "2020/02/15" }]
+        );
+        expect(value).toEqual(2);
+      });
```

Comment:
Anything wacky here with `>` and `<` on dates (that are actually strings).  Does SQLite do the right thing?

Reply:
It does as long as the format matches. We don't have that explicit test here, but I'd say the HWM tests cover that condition.